src/locales/fr/PAGES/SUMMARY.json

```
@@ -0,0 +1,19 @@
+{
+    "TITLE": "Étape 5: Résultat du rapport de l’évaluation",
+    "INTRO": "Fournir des informations supplémentaires sur l’évaluation que vous souhaitez voir figurer dans le rapport. Les résultats de l’évaluation que vous avez saisis à l’étape précédente sont inscrits ci-dessous dans la section « Résultats détaillés de l’audit » pour examen et pour vous aider à rédiger le résumé.",
```

```suggestion
    "INTRO": "Fournissez des informations supplémentaires sur l’évaluation que vous souhaitez voir figurer dans le rapport. Les résultats de l’évaluation que vous avez saisis à l’étape précédente sont inscrits ci-dessous dans la section «&nbsp;Résultats détaillés de l’audit&nbsp;» pour examen et pour vous aider à rédiger la synthèse.",
```

- See previous remark on use of infinitive
- "synthèse": consistency with the name of the "Executive summary" field below.
- Orthotypo